automerge-backend/src/backend.rs

```diff
@@ -354,6 +354,25 @@ impl Backend {
             .and_then(|index| self.history.get(*index))
     }
 
+    pub fn get_changes_added<'a>(&self, other: &'a Self) -> Vec<&'a Change> {
```

Comment:
Could we add some documentation on what this does? It's not immediately obvious to me from the name.

Reply:
Copied in the comments from the JS version.